drivers/video/ov7725.c

```diff
@@ -177,12 +177,12 @@ LOG_MODULE_REGISTER(ov7725);
 
 struct ov7725_config {
 	struct i2c_dt_spec i2c;
+#if DT_INST_NODE_HAS_PROP(0, reset_gpios)
+	struct gpio_dt_spec reset_gpio;
+#endif
```

Comment:
note: this was not coditional before

Reply:
True, but the property is optional, so see no reason not to make it conditional to match video/ov2640.c

Reply:
It might be done in a separate commit but I have no objections here.